packages/react-components/react-alert/src/stories/Alert/AlertAvatar.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+import { DismissCircleRegular } from '@fluentui/react-icons';
+
+import { Alert } from '../../index';
+
+export const AvatarProp = () => (
```

Comment:
```suggestion
export const Avatar = () => (
```

nit: The name should match the filename without the component name prefix

Reply:
I changed it due to a type error as I was importing the actual Avatar component. But that's not the case anymore so fixed it :)